packages/components/icon-button/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@spark-ui/icon-button",
+  "version": "1.0.0",
+  "description": "Icon button renders an icon within in a button",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "vite build"
+  },
+  "dependencies": {
+    "@spark-ui/button": "^1.6.0",
```

Comment:
I think we should agree to set only the major versions for our own packages, WDYT?